### PR TITLE
Issue#19ft reusable image container

### PR DESCRIPTION
**Pull request by Ochieng23**
# Description

Create reusable image containers

Fixes # 19

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshot (optional)
Screenshot from 2023-08-30 01-20-51

## How Has This Been Tested?
Rendered on the browser
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Beulah-Matt — COMMENTED** [review]
Good attempt.
This issue has however been resolved by PR #122

File: libs/elements/layout/images/src/lib/elewa-image-container/elewa-image-container.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="home-page">
+  <div *ngFor="let config of imageConfigs">
+    <h2 class="visual">
+      {{ getImageVisualisationClass(config.visualisation) }}
+    </h2>
```
Comment (LemmyMwaura) [inline libs/elements/layout/images/src/lib/elewa-image-container/elewa-image-container.component.html:5]: to add dynamic classes use ng-class directive or bind the class. interpolation doesn't work. If you were trying to just get the title then you could have achieved this directly

File: libs/features/pages/home/src/lib/main/home/home-page.component.ts [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+export class HomePageComponent {
+  imageConfigs: ImageConfig[] = [
+    {
+      title: 'Pill',
+      imageSrc:
+        'https://images.squarespace-cdn.com/content/v1/5644cf28e4b0c6c59217a904/1514921151051-YVTMSVKHFYZ1EBFNUUUB/12+Books+Black+Women+need+to+read+in+2018.jpg',
+      visualisation: ImageVisualisation.Pill,
+      maxWidth: '250px',
+    },
+    {
```
Comment (LemmyMwaura) [inline libs/features/pages/home/src/lib/main/home/home-page.component.ts:20]: No need to push these changes. They are just for your own viewing

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by LemmyMwaura** [comment]
Closing this as its already been merged, but good work

**Pull request closed without merging**